pom.xml

```diff
@@ -10,7 +10,7 @@
 	<description>A toolkit for using MATSim to simulate Mobility as a Service (MaaS)</description>
 
 	<properties>
-		<matsim.version>14.0-PR1674</matsim.version>
+		<matsim.version>14.0-SNAPSHOT</matsim.version>
```

Comment:
@luchengqi7 please use non-snapshot releases in this repo

Reply:
Opps, I merge it to the wrong place...

Reply:
I will revert the change.

I was planning to merge it into my fork. Sorry about that. 😅